scripts/jenkins/ardana/gerrit/build_test_package.py

```diff
@@ -459,6 +460,9 @@ def main():
                         help='Project in OBS that will act as the parent '
                              'project for the newly generated test project '
                              '(e.g. home:username)')
+    parser.add_argument('--buildnumber', default='NA', required=False,
+                        help='This jenkins build number for creating a unique '
```

Comment:
We should continue to maintain a clear separation between Jenkins and these backend scripts, to make it possible to run everything without requiring Jenkins. With that in mind, I think we should document this new parameter differently, so that it doesn't suggest any relationship with Jenkins. For all intents and purposes, this is just something meant to separate between build sessions.